index.js

```diff
@@ -25,11 +36,19 @@ module.exports = promisedHandlebars
  *   the promised results. This placeholder may not occur in the template or any partial. Neither
  *   my any helper generate the placeholder in to the result. Errors or wrong replacement will
  *   happen otherwise.
+ * @param {Promise} options.Promise the promise implementation to use. Defaults to global.Promise
  * @returns {Handlebars} a modified Handlebars object
  */
 function promisedHandlebars (Handlebars, options) {
   options = options || {}
   options.placeholder = options.placeholder || '\u0001'
+  if (options.Promise) {
```

Comment:
👍 
I guess in order to pass coveralls I'll have to run a test where I set global.Promise to `null`...